src/content/docs/apm/agents/c-sdk/instrumentation/distributed-tracing-c-sdk.mdx

```diff
@@ -0,0 +1,273 @@
+---
+title: 'Distributed tracing for your C services'
+tags:
+  - Agents
+  - C SDK
+  - Instrumentation
+metaDescription: 'For the New Relic C SDK: how to implement distributed tracing across your C language applications and services.'
+redirects:
+  - /docs/agents/c-sdk/instrumentation/enable-distributed-tracing-your-c-applications
+  - /docs/apm/agents/c-sdk/instrumentation/enable-distributed-tracing-your-c-applications
+---
+
+import logsInContextTraces from 'images/logs-in-context-traces.png'
+
+import clogo from 'images/clogo.png'
+
+Distributed tracing allows you to see the entire journey of your requests throughout a [distributed system](/docs/distributed-tracing/concepts/introduction-distributed-tracing). The C SDK supports [standard distributed tracing](/docs/distributed-tracing/concepts/how-new-relic-distributed-tracing-works/#head-based) with head-based sampling. It doesn't support the advanced feature called Infinite Tracing or W3C Trace context.
+
+The C SDK requires you to manually instrument your C services, unlike the auto-instrumentation of the other New Relic agents. This means you need to add some lines to your code to use the C SDK, and then you add an additional configuration to enable distributed tracing.
+
+Here's how to get started:
+
+* [Instrument a sample program (recommended):](#quick-start-apm) A fast way to get started
+* [Manual instrumentation:](#manual-instrumentation) Tips for additional instrumentation
+
+<Callout variant="tip">
+  If you want to get more background before getting started, check out these topics:
+  * [How span sampling works](/docs/understand-dependencies/distributed-tracing/get-started/how-new-relic-distributed-tracing-works#sampling) explains distributed tracing options.
+  * [Impacts to APM](/docs/apm/distributed-tracing/getting-started/transition-guide-distributed-tracing) tells you what to expect if you are a current APM user but haven't set up distributed tracing.
+</Callout>
+
+## Instrument a sample program (recommended): [#quick-start-apm]
+
+This is a quick way to see how to install the C SDK and instrument a service with [standard distributed tracing](/docs/distributed-tracing/concepts/how-new-relic-distributed-tracing-works/#head-based). 
+
+<Callout variant="tip">
+The advanced option called Infinite Tracing isn't available with C SDK.
+</Callout>
+
+You'll need a New Relic account to set up distributed tracing. If you don't already have one, you can quickly [create a free account](https://newrelic.com/signup). Note that this link will take you to another site to complete the sign-up, but you can return here and follow the setup steps below:
+
```

Comment:
Please add some text here that before proceeding, please verify your system meets the requirements and that you are able to compile your code with the C SDK and that the daemon is started as outlined in the instructions here:
https://docs.newrelic.com/docs/apm/agents/c-sdk/install-configure/install-c-sdk-compile-link-your-code

These steps are actually important, because it's not just adding the code as in the snippets provided below, but the header file needs to be compiled with their code as well, and then the daemon needs to be started.